.github/workflows/darwin.yaml

```diff
@@ -118,6 +118,20 @@ jobs:
               # should change the deployment target version instead?
               run: xcodebuild -target "Matter" -sdk macosx OTHER_CFLAGS='${inherited} -Werror -Wconversion -Wno-unguarded-availability-new'
               working-directory: src/darwin/Framework
+            - name: Run macOS DFT Build Debug
+              timeout-minutes: 50
```

Comment:
This is a significant amount of time to add to CI.  How would you feel about just using xcodebuild to build DFT in `.github/workflows/darwin-tests.yaml`, where we are building it anyway?

We really need to get to a point where we have only one supported way to build darwin-framework-tool....

Reply:
Agree!